cns/restserver/restserver.go

```diff
@@ -292,6 +292,7 @@ func (service *HTTPRestService) Init(config *common.ServiceConfig) error {
 	listener.AddHandler(cns.NetworkContainersURLPath, service.getOrRefreshNetworkContainers)
 	listener.AddHandler(cns.GetHomeAz, service.getHomeAz)
 	listener.AddHandler(cns.EndpointPath, service.EndpointHandlerAPI)
+	listener.AddHandler(cns.GetNCList, service.nmAgentNCListHandler)
```

Comment:
Evan had some sort of concern about exposing APIs to the AKS side (if it is not needed) and that's why I had added the below statement where we add API only for direct channel mode. Should we do the same here?

Reply:
I made the requested change after going through your conversation with Evan. Thanks

Reply:
thanks for remembering this